packages/ng/src/app/select/option/select-option-feeder.model.ts

```diff
@@ -0,0 +1,25 @@
+export interface ISelectOptionFeeder<T> {
+
+	/**
+	 * @returns true if the focus is on the field
+	 */
+	hasFocus(): boolean;
+
+	/**
+	 * Called when the popup of option is open
+	 */
+	open(): void;
+
+	/**
+	 * Register key events
+	 * @param callback
+	 */
+	registerKeyevent(callback: (event: KeyboardEvent) => void): void;
```

Comment:
wot wot ?

Reply:
As The option feeder manipulate the focus, and the elements, we have to get back the control of key events => up, down, enter, .....